src/NzbDrone.Api/Config/IndexerConfigModule.cs

```diff
@@ -13,6 +13,9 @@ public IndexerConfigModule(IConfigService configService)
             SharedValidator.RuleFor(c => c.MinimumAge)
                            .GreaterThanOrEqualTo(0);
 
+            SharedValidator.RuleFor(c => c.MaximumSize)
+                            .GreaterThanOrEqualTo(0);
```

Comment:
Nitpick: one space too much.
Wouldn't have commented on it if you didn't have to change something else. :smile: